contentcuration/contentcuration/frontend/shared/views/__tests__/appIcon.spec.js

```diff
@@ -0,0 +1,40 @@
+import { mount } from '@vue/test-utils';
+import Vue from 'vue';
+import Vuetify from 'vuetify';
+import AppIcon from '../AppIcon.vue';
+
+Vue.use(Vuetify);
```

Comment:
This is redundant to global Jest setup https://github.com/bjester/studio/blob/channel-settings/jest_config/setup.js#L4. I've just tried to run new specs without this line and seems to work fine.

Reply:
Nice catch. This is copy pasta, so I'll clean it up.